repos/fdbt-site/src/pages/salesConfirmation.tsx

```diff
@@ -31,6 +31,9 @@ interface TicketDating {
     startDefault: boolean;
     endDefault: boolean;
 }
+export const formatSalesOfferPackageEnumerations = (enumerations: string[]): string[] => {
+    return enumerations.map((enumeration) => ` ` + upperFirst(lowerCase(enumeration.replace(/([A-Z])/g, ' $1'))));
```

Comment:
I think instead of attempting to convert the Netex to a display string we should use the actual display values from the SOP screen. There are some with don't match the simple mapping i.e electronic_document to mapped to Digital

Reply:
![image](https://user-images.githubusercontent.com/77591337/126314170-62726905-e2f4-49a7-969b-85e9d20dc413.png)
Could I just add an if statement for electronic document since that's the only exception?

I ask because I'm still wrapping my head around how the SOP page get's the vaules and adding the exception might be quicker

Reply:
![image](https://user-images.githubusercontent.com/77591337/126314665-52d258e9-fa3f-45ef-a1fe-d4bfc17952f2.png)